### ISSUE TITLE
renderRightButton example?

### ISSUE THREAD

**Issue by vugman** [aksonov/react-native-router-flux#100]
Hey, couldn't find any documentation or example on how to use renderRightButton and renderRightButtonStyle.

Could you please add some?

**Comment by aksonov** [comment]
It is in TODO list. For any missed docs, just refer to exNavigator, this component just passes these properties to exNavigator component

**Comment by vugman** [comment]
I've tried that... all I get is the following error:
<img width="357" alt="screen shot 2015-12-31 at 11 15 14" src="https://cloud.githubusercontent.com/assets/11426508/12063013/d6a9faf2-afaf-11e5-857b-6d110d18c9c2.png">

**Comment by aksonov** [comment]
You should pass _function_ which returns react component.

**Comment by vugman** [comment]
Here's my code:

```
RightButton.js
....
var RightButton = React.createClass({
  render: function() {
      return (
          <View stlye={styles.rightButton}>
              <Button style={{color: 'green'}}>
                 +
              </Button>
          </View>
      );
  }
....
})
```

```
App.js
....
var RightButton = require('./components/RightButton');
....
<Route name="something" component={Something} renderRightButton={RightButton} wrapRouter={false} title={'Requests'} hideNavBar={false}/>
....
})
```

What am I missing?

**Comment by ippy04** [comment]
@vugman I believe you're passing in a RN component instead of a function that creates a RN component, i.e. sbould be something like this: 

var createRightButton = function() {
      return (
          <View stlye={styles.rightButton}>
              <Button style={{color: 'green'}}>
                 +
              </Button>
          </View>
      );

<Route name="something" renderRightButton={createRightButton}>

**Comment by vugman** [comment]
Ohhh thanks @ippy04 thanks for the help!

**Comment by andreasbhansen** [comment]
@aksonov Does this still work in 2.0.2? After updating, the button which used to be in the navBar is no longer there.

**Comment by aksonov** [comment]
@andreasbhansen Thanks, should be fixed in 2.0.3

**Comment by andreasbhansen** [comment]
Thanks, @aksonov. Works fine now!
